Cabal/src/Distribution/Simple/Flag.hs

```diff
@@ -46,18 +46,21 @@ import Prelude ()
 --
 -- 1. list flags eg
 --
--- > --ghc-option=foo --ghc-option=bar
+--   > --ghc-option=foo --ghc-option=bar
 --
--- gives us all the values ["foo", "bar"]
+--   gives us all the values ["foo", "bar"]
 --
 -- 2. singular value flags, eg:
 --
--- > --enable-foo --disable-foo
+--   > --enable-foo --disable-foo
 --
--- gives us Just False
--- So this Flag type is for the latter singular kind of flag.
+--   gives us Just False
```

Comment:
This one is an example of pre-existing missing markup, which is why I didn't say anything about it originally,

Reply:
@geekosaur

I'll put that on my to-do list.

I'm more focusing on keeping a tempo up, and I guess I'll try to focus more on editing existing comments as well.